app/scripts/controllers/dashboard.coffee

```diff
@@ -158,6 +158,7 @@ angular.module('edudashAppCtrl').controller 'DashboardCtrl', [
 
         $scope.setSchool = (item, model, showAllSchools) ->
             $scope.selectedSchool = item
+            $scope.selectedSchool['pupil-teacher-ration'] = 56
```

Comment:
Is this for debug until we have real data? Looking at `item` here, there is a `item.pt_ratio` available that might be ok to use.